Wangscape/Wangscape.vcxproj.filters.orig

```diff
@@ -0,0 +1,191 @@
+    <ClInclude Include="noise\module\codecs\CurveWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\CylindersWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\DisplaceWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\ExponentWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\InvertWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\MaxWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\MinWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\MultiplyWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\PowerWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\RidgedMultiWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\RotatePointWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\ScaleBiasWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\SelectWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\SpheresWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\TerraceWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\TranslatePointWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\GradientWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\CornerCombinerBaseWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\ExpWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\PowWrapperCodec.h" />
+    <ClInclude Include="noise\module\codecs\NormLPQWrapperCodec.h" />
+    <ClInclude Include="noise\module\Forward.h" />
+    <ClInclude Include="noise\module\codecs\ForwardWrapperCodec.h" />
+    <ClInclude Include="tilegen\partition\TilePartitionerNoise.h" />
+    <ClInclude Include="noise\module\codecs\QuadrantSelectorWrapperCodec.h" />
+    <ClInclude Include="noise\module\QuadrantSelector.h" />
+    <ClInclude Include="noise\ModuleGroup.h" />
+    <ClInclude Include="codecs\TerrainModuleGroupLocationCodec.h" />
+    <ClInclude Include="TerrainModuleGroupLocation.h" />
+    <ClInclude Include="BorderModuleGroupLocation.h" />
+    <ClInclude Include="codecs\BorderModuleGroupLocationCodec.h" />
+    <ClInclude Include="noise\codecs\EncodedModuleGroupCodec.h" />
+    <ClInclude Include="noise\module\UniformTriple.h" />
+    <ClInclude Include="noise\module\codecs\UniformTripleCodec.h" />
+    <ClInclude Include="noise\module\codecs\ScalePointWrapperCodec.h" />
+    <ClInclude Include="noise\ModuleID.h" />
+    <ClInclude Include="noise\module\WangscapeModules.h" />
+    <ClInclude Include="noise\WangscapeNoise.h" />
+    <ClInclude Include="noise\WangscapeNoiseCodecs.h" />
+    <ClInclude Include="noise\ModuleGroupContainer.h" />
+    <ClInclude Include="OptionsPaths.h" />
+    <ClInclude Include="codecs\Vector2Codec.h" />
+<<<<<<< Updated upstream
+=======
+    <ClInclude Include="DebugWriter.h" />
+    <ClInclude Include="tilegen\CornerTerrains.h" />
+>>>>>>> Stashed changes
```

Comment:
Yep, this is noise from kdiff3. I don't like that it leaves these files around. I'll add `*.orig` to `.gitignore`.